examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>there is also a version with 3 parties + 1 crypto provider I think in SecureNN</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Oh yes I made a mistake, in the paper they actually do consider 3 parties + 1 crypto provider. </p>

Reply:
<p>They describe the system as a 3PC-protocol, and the algorithms always consider two parties holding the secret shares (referred to as P0, and P1) and one party generating the random primitives (P2).</p><p> </p><p>Do you have some resource pointing to the 4PC-version which I should include or should I simply state it? </p><p> </p>

Reply:
Maybe you can just mention that the 4PC version exist without detailing yeah, maybe that's not the most important par of SecureNN (and we don't implement it!)

Reply:
Sounds good, I'll do that!